src/styles/_variables.scss

```diff
@@ -52,6 +52,8 @@
     --mynah-bg-gradient-mid: #538bf7;
     --mynah-bg-gradient-start: #301673;
     --mynah-bg-gradient-end: #6e3bf1;
+    --mynah-bg-gradient-next: #da8af7;
```

Comment:
What are these used for?

Reply:
These are 2 additional values for the background, we need to validate with the UX team first, but i also don't want to add more colors for the gradient. Until we have a validation from the UX team, they'll stay.